pysmt/typing.py

```diff
@@ -279,12 +279,25 @@ def __ne__(self, other):
     def __hash__(self):
         return self._hash
 
+class StringType(PySMTType):
+    def __init__(self):
+        PySMTType.__init__(self, type_id = 5)
+    
+    def is_string_type(self):
+        return True
+    def as_smtlib(self, funstyle=True):
+        if funstyle:
+            return "() String"
+        else:
+            return "String"
+    def __str__(self):
+        return "String"
 
 # Singletons for the basic types
 BOOL = BooleanType()
 REAL = RealType()
 INT = IntType()
 
```

Comment:
Probably the error our CI signals is here:

you need to add a line
```python
STRING = StringType()
```
Otherwise the constant 'STRING' in line 302 is undefined.